### DISCUSSION TITLE
Nativesvttk

### DISCUSSION THREAD

**Discussion by JTSJohnny** [Akascape/tkinter-toolkit#14] [Add this]
dependices: darkdetect
It is a tkinter app

**Comment by thisal-d** [comment]
> dependices: darkdetect It is a tkinter app

are you talking about [this](https://pypi.org/project/nativesvttk/) ?
where is the github reposotory for Nativesvttk ? I couldnt find it !

**Comment by JTSJohnny** [comment]
It's a codespace! I can send u the source.
________________________________

**Reply by thisal-d** [reply]
> It's a codespace! I can send u the source.
> […](#)
> ________________________________ From: Thisal Dilmith ***@***.***> Sent: Saturday, May 25, 2024 4:08 PM To: Akascape/tkinter-toolkit ***@***.***> Cc: jhi2 ***@***.***>; Author ***@***.***> Subject: Re: [Akascape/tkinter-toolkit] Nativesvttk (Discussion #14) dependices: darkdetect It is a tkinter app are you talking about this<https://pypi.org/project/nativesvttk/> ? where is the github reposotory for Nativesvttk ? I couldnt find it ! — Reply to this email directly, view it on GitHub<[#14 (comment)](https://github.com/Akascape/tkinter-toolkit/discussions/14#discussioncomment-9558511)>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/A3V45HBXGAMSSUD5LNMOUELZEEKYDAVCNFSM6AAAAABIJGHIN6VHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4TKNJYGUYTC>. You are receiving this because you authored the thread.Message ID: ***@***.***>

send it :). it will helpfull

**Comment by JTSJohnny** [comment]
ok
________________________________

**Comment by JTSJohnny** [comment]
I have the source. I have a lot of other files like the README and the upload script and assets for manual tcl but we don't need those.
________________________________

**Reply by JTSJohnny** [reply]
Hey I attached it by email. It did not work Let me upload it to my site.

**Reply by thisal-d** [reply]
at least where is the doumentation ?

**Reply by thisal-d** [reply]
> Hey I attached it by email. It did not work Let me upload it to my site.

sure.

**Reply by JTSJohnny** [reply]
> at least where is the doumentation ?

The docs? On Windows, just run native_theme(root). The rest is like sv_ttk. and there is a compatability check.

**Reply by JTSJohnny** [reply]
> Hey I attached it by email. It did not work Let me upload it to my site.

the link: https://internalfiles.johnnytech.net/githubfiles/nsv/nsv3.zip

**Reply by thisal-d** [reply]
Is the zip file you sent password protected ? i can't extract it.

**Reply by JTSJohnny** [reply]
> Is the zip file you sent password protected ? i can't extract it.

The password is `jhi2`.

**Reply by JTSJohnny** [reply]
Any changes yet?

**Reply by thisal-d** [reply]
i did a pull request. he will check it.

**Reply by thisal-d** [reply]
https://github.com/Akascape/tkinter-toolkit/pull/15
